lib/still/compiler/traverse.ex

```diff
@@ -14,37 +14,30 @@ defmodule Still.Compiler.Traverse do
     with true <- File.dir?(get_input_path()),
          _ <- File.rmdir(get_output_path()),
          :ok <- File.mkdir_p(get_output_path()) do
-      do_run()
+      compilable_files()
+      |> Enum.map(&CompilationStage.compile/1)
```

Comment:
I guess we don't need the `Enum.map` in here right?

Reply:
I guess we don't. I didn't notice at first that `compile/1` accepted a list